sdk/dotnet/Ssm/Document.cs

```diff
@@ -115,7 +115,7 @@ public partial class Document : Pulumi.CustomResource
         /// Additional Permissions to attach to the document. See Permissions below for details.
         /// </summary>
         [Output("permissions")]
-        public Output<Outputs.DocumentPermissions?> Permissions { get; private set; } = null!;
+        public Output<ImmutableDictionary<string, string>?> Permissions { get; private set; } = null!;
```

Comment:
Another one similar

Reply:
Similar change - changed from list to map

https://github.com/terraform-providers/terraform-provider-aws/commit/877b365779abcc5d5b4c3e110999003d3fe7c096#diff-0b148b55bba39feb06b753e488b56fcf